plugins/search/locallang.xml

```diff
@@ -68,6 +71,9 @@
 			<label index="tt_content.pi_flexform.facets">Diese Facetten anzeigen</label>
 			<label index="tt_content.pi_flexform.limitFacets">Maximale Facettenwerte</label>
 			<label index="tt_content.pi_flexform.resetFacets">Facetten zurücksetzen erlauben</label>
+			<label index="tt_content.pi_flexform.sortingFacets">Facetten Sortierung</label>
+			<label index="tt_content.pi_flexform.sortingFacets.count">Count</label>
+			<label index="tt_content.pi_flexform.sortingFacets.index">Index</label>			
```

Comment:
`Count` and `Index` should be replaced by German translations. I don't even know, what sorting by `index` means. :)

Reply:
Good point :)